examples/atlas-backup-snapshot-restore-job/point-in-time/versions.tf

```diff
@@ -1,9 +1,8 @@
 terraform {
   required_providers {
     mongodbatlas = {
-      source  = "mongodb/mongodbatlas"
-      version = "~> 1.4.6"
```

Comment:
this is great fix @andreaangiolillo, in future for all examples we should just default to pulling latest version of Terraform Atlas Provider